apps/antalmanac/tests/download-ics.test.ts

```diff
@@ -83,13 +66,25 @@ describe('download-ics', () => {
                 startOutputType: 'local',
                 endOutputType: 'local',
                 title: 'placeholderDeptCode placeholderCourseNumber Final Exam',
-                description: 'Final Exam for placeholderCourseTitle',
+                description: 'Final Exam for placeholderSectionType placeholderCourseTitle',
                 start: [2023, 3, 3, 1, 2],
                 end: [2023, 3, 3, 3, 4],
             },
+            {
+                productId: 'antalmanac/ics',
+                startOutputType: 'local',
+                endOutputType: 'local',
+                title: 'placeholderCustomEventTitle',
+                // TODO: Add location to custom events, waiting for https://github.com/icssc/AntAlmanac/issues/249
+                // location: `${location.building} ${location.room}`,
+                start: [2023, 9, 29, 1, 2],
+                end: [2023, 9, 29, 3, 4],
+                recurrenceRule: 'FREQ=WEEKLY;BYDAY=MO,WE,FR;INTERVAL=1;COUNT=31',
+            },
         ];
 
         const result = getEventsFromCourses(courses);
+        console.log(result);
```

Comment:
If we're asserting this is equal to the expected one, I don't think we need to print it out here.

Reply:
I'll make it a code suggestion 👍